lib/meilisearch/index.rb

```diff
@@ -461,5 +461,22 @@ def update_faceting(faceting_attributes)
     def reset_faceting
       http_delete("/indexes/#{@uid}/settings/faceting")
     end
+
+    ### SETTINGS - DICTIONARY
+
+    def dictionary
+      http_get("/indexes/#{@uid}/settings/dictionary")
+    end
+    alias get_dictionary dictionary
```

Comment:
Same thing as the other PR, let's remove these aliases :)

Reply:
Same as the other PR, I am assuming you mean the getter only, and leaving the setter alias.